auth_otp/services/auth_service.py

```
@@ -0,0 +1,437 @@
+            user = User.objects.create_user(
+                username=phone_number,
+                user_type=user_type,
+                is_active=True
+            )
+            is_new = True
+            logger.info(f"New user created: {phone_number}, type: {user_type}")
+        
+        return user, is_new
+    
+    @staticmethod
+    def generate_tokens(user: User) -> Dict[str, any]:
```

Ruff (0.12.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)